Penteract/Assets/Scripts/AIMeleeGrunt.cpp

```diff
@@ -418,6 +437,7 @@ void AIMeleeGrunt::EnableBlastPushBack() {
 	if (state != AIState::START && state != AIState::SPAWN && state != AIState::DEATH) {
 		gruntCharacter.beingPushed = true;
 		state = AIState::PUSHED;
+		particlesPush->PlayChildParticles();
```

Comment:
Check nullity. In the start you check the nullity of the aux local variable, but at this point this pointer could be null